OpenEdXMobile/src/main/java/org/edx/mobile/http/notifications/OverlayErrorNotification.java

```diff
@@ -0,0 +1,163 @@
+package org.edx.mobile.http.notifications;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.StringRes;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+import android.widget.Button;
+import android.widget.FrameLayout;
+import android.widget.TextView;
+
+import com.joanzapata.iconify.Icon;
+import com.joanzapata.iconify.widget.IconImageView;
+
+import org.edx.mobile.R;
+
+import static android.view.View.GONE;
+import static android.view.View.VISIBLE;
+import static android.view.ViewGroup.LayoutParams.MATCH_PARENT;
+
+/**
+ * An overlay notification error message on top of the content area.
```

Comment:
`which could also possess error icon and action button.` is an implementation detail and shouldn't be included in the description.

But yeah the description could be improved like:
`An full screen error notification on top of the content area.`